test/lib/schematics/schematic.option.spec.ts

```diff
@@ -38,19 +38,19 @@ describe('Schematic Option', () => {
       input: 'myApp',
       expected: 'my-app',
     },
-    {
+    /*{
       description: 'should allow underscore string option value name',
       option: 'name',
       input: 'my_app',
       expected: 'my_app',
-    },
+    },*/
     {
       description: 'should manage classified string option value name',
       option: 'name',
       input: 'MyApp',
       expected: 'my-app',
     },
-    {
+    /*{
       description: 'should manage parenthesis string option value name',
```

Comment:
Atm we don't really respect this. In `package.json` we have the right name `my-(app)`, but at least on linux the folder is actually called `./'my-(app)'` (note the dashes). Same for the `[` character